ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -650,34 +673,28 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
               <div style={{display: 'flex', flexDirection: 'column'}}>
               <div style={{display: 'flex', alignItems: 'flex-end'}}>
                 <div style={styles.previewDataHeader}>
-                  Preview Data Set
+                  <div>
+                    <CircleWithText text={4} width='23.78px' height='23.78px'
+                      style={{marignTop: '0.3rem', fill: colorWithWhiteness(colors.primary, 0.5)}}/>
+                  </div>
+                  <label style={{marginLeft: '0.5rem', color: colors.primary}}>
+                    Preview Data Set
```

Comment:
Existing issue, but the mocks say "Dataset" in most places, this should match. This file is not self-consistent on this